tests/profiling/collector/test_stack_asyncio.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import collections
+
+from ddtrace.profiling import profiler
+from ddtrace.profiling.collector import stack_event
+
+
+def test_asyncio(tmp_path, monkeypatch) -> None:
+    sleep_time = 0.2
+    sleep_times = 5
+
+    async def stuff() -> None:
+        await asyncio.sleep(sleep_time)
+
+    async def hello() -> None:
+        asyncio.create_task(stuff(), name="sleep 1")
+        asyncio.create_task(stuff(), name="sleep 2")
+        for _ in range(sleep_times):
+            await stuff()
+
+    monkeypatch.setenv("DD_PROFILING_CAPTURE_PCT", "100")
+    monkeypatch.setenv("DD_PROFILING_OUTPUT_PPROF", str(tmp_path / "pprof"))
+    # start a complete profiler so asyncio policy is setup
+    p = profiler.Profiler()
+    p.start()
+    asyncio.run(hello())
+    events = p._profiler._recorder.reset()
+    p.stop()
+
+    wall_time_ns = collections.defaultdict(lambda: 0)
+
+    for event in events[stack_event.StackSampleEvent]:
+
+        wall_time_ns[event.task_name] += event.wall_time_ns
+
+        if event.task_name == "Task-1":
+            assert event.thread_name == "MainThread"
+            assert event.frames == [(__file__, 19, "hello")]
+            assert event.nframes == 1
+        elif event.task_name == "sleep 1":
+            assert event.thread_name == "MainThread"
+            assert event.frames == [(__file__, 13, "stuff")]
+            assert event.nframes == 1
+        elif event.task_name == "sleep 2":
+            assert event.thread_name == "MainThread"
+            assert event.frames == [(__file__, 13, "stuff")]
+            assert event.nframes == 1
```

Comment:
Is there any assertions we do to be sure we got all of these at least once?

Reply:
Good idea.

Reply:
Actually we don't since there's a check for wall time that makes sure we have those task seen once :)